content/about/security-vulnerabilities/index.md

```diff
@@ -43,15 +45,38 @@ projects. We only share the information as needed to fix the issue.
 We keep the reporter updated as the status of the security issue moves
 from `triaged`, to `identified fix`, to `release planning`.
 
-## Public disclosure timing
-
-The Istio security team and the bug submitter negotiate a public
-disclosure date between them. We prefer to fully disclose the bug as
-soon as possible once a user mitigation is available.
-We consider reasonable to delay disclosure when the bug or the fix is
-not yet fully understood, the solution is not well-tested, or for
-vendor coordination. The time frame for disclosure is from immediate,
-especially if the bug is known publicly already, to a few weeks.
-As a basic default, we expect the report date and the disclosure date
-to be on the order of seven days apart. The Istio security team holds
-the final say on setting a disclosure date.
+## Fixing the issue
+
+Once a security vulnerability has been fully characterized, a fix is developed by the Istio team.
+The development and testing for the fix happens in a private GitHub repository in order to prevent
+premature disclosure of the vulnerability.
+
+## Early disclosure
+
+Before vulnerabilities are disclosed to the public, there is an early private disclosure to a small set of Istio partners.
+This is to enable partners that distribute Istio binaries to prepare to distribute the fix.
+
+Early disclosure happens three working days prior to full public disclosure.
+
+Please fill out the [Early Security Vulnerability Disclosure](https://github.com/istio/community/issues/new?template=early-disclosure-request.md) form to request being
+added to the early disclosure mailing list.
+
+## Public disclosure
+
+On the day chosen for public disclosure, a sequence of activities takes place as quickly as possible:
+
+- Changes are merged from the private GitHub repository holding the fix into the appropriate set of public
+branches.
+
+- Release engineers ensure all necessary binaries are promptly built and published.
+
+- Once the binaries are available, an announcement is sent out on the following channels:
+
+    - The [Istio blog](https://istio.io/blog)
+    - The [Announcements](https://discuss.istio.io/c/announcements) category on discuss.istio.io
+    - The [Istio Twitter feed](https://twitter.cqom/IstioMesh)
+    - The [#announcement channel on Slack](https://istio.slack.com/messages/CFXS256EQ/)
+
+As much as possible this announcement should be actionable, and include any mitigating steps users can take prior to
+upgrading to a fixed version. The recommended target time for these announcements is 4pm UTC from Monday to Thursday.
```

Comment:
I know it is a strange request, but I'd recommend 24 hour format for UTC.  Specified as 4pm UTC is odd, and may not match the UTC specification.